lldb/unittests/TestingSupport/TestUtilities.h

```diff
@@ -30,6 +30,10 @@
   }
 
 namespace lldb_private {
+
+/// Returns a pretty printed json string of a `llvm::json::Value`.
+std::string pprint(const llvm::json::Value &E);
```

Comment:
I'd call this `PrettyPrint`, makes it immediately obvious what it does. 

```suggestion
std::string PrettyPrint(const llvm::json::Value &E);
```